src/main/java/net/dv8tion/jda/api/entities/User.java

```diff
@@ -225,4 +225,70 @@ default String getEffectiveAvatarUrl()
      */
     @Nonnull
     JDA getJDA();
+
+    /**
+     * Represents the bit offsets used by Discord for public flags
+     */
+    enum Flags{
+        NONE(                      0, "None"),
+        STAFF(                     1, "Discord Employee"),
+        PARTNER(                   2, "Discord Partner"),
+        HYPESQUAD(                 4, "HypeSquad Events"),
+        BUG_HUNTER_LEVEL_1(        8, "Bug Hunter Level 1"),
+        // Bravery
+        HYPESQUAD_ONLINE_HOUSE_1( 64, "HypeSquad Online House Bravery"),
+        // Brilliance
+        HYPESQUAD_ONLINE_HOUSE_2(128, "HypeSquad Online House Brilliance"),
+        // Balance
+        HYPESQUAD_ONLINE_HOUSE_3(256, "HypeSquad Online House Balance"),
+        PREMIUM_EARLY_SUPPORTER( 512, "Early Supporter"),
+        TEAM_USER(              1024, "Team User"),
+        SYSTEM(                 4096, "System User"),
+        BUG_HUNTER_LEVEL_2(    16384, "Bug Hunter Level 2"),
+        VERIFIED_BOT(          65536, "Verified Bot"),
+        VERIFIED_DEVELOPER(   131072, "Verified Developer");
+        
+        private final int offset;
+        private final long raw;
+        private final String name;
+        
+        Flags(int offset, @Nonnull String name)
+        {
+            this.offset = offset;
+            this.raw = 1 << offset;
```

Comment:
This value here is incorrect, you are already passing the raw offset as the first paramater;
Example: the "raw" varaible for NONE will be 1, 2 for STAFF, 4 fpr HYPESQUAD.

To fix this issue you should change each Enums value to the offset instead of the raw bit shifted value

Reply:
I've fixed it in a PR [here](https://github.com/Andre601/JDA/pull/3)

The "Flags" enum should probably be renamed to UserFlags as well

Reply:
@boomboompower Agreed. UserFlag would be more readable